### ISSUE TITLE
Metrics: Target page is unknown without TracingStartedInPage trace events

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/lighthouse#521]
A few of our metrics use `TracingStartedInPage` to identify the renderer process we want to inspect.
When ChromeDriver collects a trace, it connects to the browser instead of a renderer target, and therefor these events don't show up in the trace.

We need a fallback path if TracingStartedInPage isn't found in the trace.

> you can resort to finding a process by the process name (which, in case of renderer, should include a URL).

We can also look for other events that indicate a renderer of interest is doing the right work.

cc @michaelgerakis

**Comment by paulirish** [comment]
We have at least two more signals we can use. Neither is a sure thing, but for a fairly controlled environment, it shouldn't be a problem.

### 1. page `<title>`

The trace has `process_labels` metadata events like so:
```json
  {
    "pid": 1480,
    "tid": 17447,
    "cat": "__metadata",
    "name": "process_labels",
    "args": { "labels": "Lighthouse report: https://www.selioapp.com/" }},
```

The this label is set in [`RenderFrameImpl::didReceiveTitle` here](https://cs.chromium.org/chromium/src/content/renderer/render_frame_impl.cc?dr=C&q=f:render_frame_impl+UpdateProcessLabel&sq=package:chromium) and is updated each time the page title changes. 

Caveats: If the renderer is navigated to about:blank or the dinosaur page at the end of the run, this would have problems, but otherwise it will match up against the actual `document.title` fairly well.  Pretty sure I've seen some encoding barfs, but not sure if the problem was in the trace.

### 2. Count of trace events

The renderer's that we're reloading are by far the most active renderer threads in the trace. 

Caveats: If our trace was *only* scrolling the page and not reloading it, and a background tab was a heavy webapp doing a lot of work, it's possible we'd get it wrong. However the background tab situation is unlikely when we're using chromedriver (can be a clean chrome profile), and for our extension usecase, we are safely relying on TracingStartedInPage.


-----

Here's a test with one of our traces. The "Selio Lite" renderer with `5211` events is far and away the most voluminous renderer thread, so it's our candidate.

![image](https://cloud.githubusercontent.com/assets/39191/17111306/fd233452-526e-11e6-8f13-7f961acfbbb0.png)

If no one has any objections, I'll start getting this in place.

**Comment by paulirish** [comment]
fixed by #538
